ipyvolume/pylab.py

```diff
@@ -37,7 +37,7 @@ def _docsubst(f):
     "color2d"] = "color for each point/vertex string format, examples for red:'red', '#f00', '#ff0000' or 'rgb(1,0,0), or rgb array of shape (2, N, 3) or (S, 2, N, 3)"
 _doc_snippets[
     "size"] = "float representing the size of the glyph in percentage of the viewport, where 100 is the full size of the viewport"
-_doc_snippets["marker"] = "name of the marker, options are: 'arrow', 'box', 'diamond', 'sphere'"
+_doc_snippets["marker"] = "name of the marker, options are: 'arrow', 'box', 'diamond', 'sphere', 'point_2d', 'tri_2d'"
```

Comment:
Yes, good choice, if we'd prefix it, we could never turn them into valid Python variable names, since we might consider doing an Enum in the future.